zulip_bots/zulip_bots/bots/flock/flock.py

```diff
@@ -18,6 +18,19 @@ def find_recipient_id(res: str, recipient_name: str) -> str:
         if recipient_name == obj['firstName']:
             return obj['id']
 
+# Male request to given flock URL and return two-element tuple
```

Comment:
Typo: `s/Male/Make`. Also, return *a* two-element tuple 😄  .

Reply:
@aero31aero Fixed typo in related commit.